### PR TITLE
doc: fix broken links in openstack topology exampples

### PR DESCRIPTION
**Pull request by lukas-bednar**
Signed-off-by: Lukas Bednar <lbednar@redhat.com>

### CODE REVIEW

File: docs/source/openstack.rst [hunk 1]
```diff
@@ -65,23 +65,23 @@ os_obj
 
 OpenStack Object Storage can be provisioned using this resource.
 
-* :docs1.5:`Topology Example <workspaces/topologies/os-obj-new.yml>`
+* :docs1.5:`Topology Example <workspace/topologies/os-obj-new.yml>`
```
Comment (14rcole) [inline docs/source/openstack.rst]: We're trying to do away with the `workspace` directory.  Can you link to `workspaces/openstack/topologies/os-obj-new.yml`?  And also do the same with the other changes here?

Comment (lukas-bednar) [reply docs/source/openstack.rst]: Done, there was no os_keypair.yml in the new location, so I just copied it from old directory, I hope it is ok.

**Comment by 14rcole** [comment]
@lukas-bednar if you add pytest to the array at line 43 of `setup.yml` it'll resolve the unit test issue in jenkins.

```
43         'tests': ["nose", "mock", "coverage", "flake8", "pytest"],
```

**Comment by samvarankashyap** [comment]
@lukas-bednar needs rebase.

**Comment by samvarankashyap** [comment]
needs rebase

**Comment by samvarankashyap** [comment]
@lukas-bednar Needs rebase

**Comment by samvarankashyap** [comment]
Merging this PR since its no way related to tests failures

**Comment by samvarankashyap** [comment]
[merge]

**Pull request merged**